### ISSUE TITLE
Use the qualified reference > Allow to import function instead of backslash

### ISSUE THREAD

**Issue by ruudk** [kalessil/phpinspectionsea#1079]
First of all, your plugin rocks! Saves my ass every day. Just put in an order for Ultimate. Thanks!

| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | Feature request |
| Plugin         | Php Inspections (EA Extended) v3.0.7 |
| Language level | PHP 7.2                        |

Current behaviour (description/screenshot:)
--
<img width="522" alt="screenshot 2018-10-23 at 15 02 37" src="https://user-images.githubusercontent.com/104180/47362445-b3a2c580-d6d4-11e8-8e07-2048aa2fa561.png">
<img width="472" alt="screenshot 2018-10-23 at 15 02 51" src="https://user-images.githubusercontent.com/104180/47362458-ba313d00-d6d4-11e8-9ecb-207ba16d3d24.png">

```php
if (3 !== \count($matches)) {
    return null;
}
```

Expected behaviour
--
```php
use function count;

if (3 !== count($matches)) {
    return null;
}
```

Notes
--
I really like to import my functions instead of adding that ugly backslash. Would be great if the plugin offered an option, either just put the backslash, or import the function.

**Comment by kalessil** [comment]
Thank for a great feedback @ruudk, since we are going into the next stabilization round, I think it's time to finally implement this =)
